cli.js

```diff
@@ -65,6 +67,12 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
         describe: 'Function name',
         type: 'string',
       })
+      .option('executionModel', {
```

Comment:
I don't see why this is required during create.
We should add this to the template function if it's not there.

Reply:
Because it's a value configured in the yml. The only two comparable features are 

1. Runtime selection (forcefully required at `create` time)
2. Secrets, not written by the template and not required by the user ever